src/Export/Scripts/uniqueText.lua

```diff
@@ -0,0 +1,77 @@
+if not table.containsId then
+	dofile("Scripts/mods.lua")
+end
+
+local itemTypes = {
```

Comment:
Since there is data that is shared by multiple scripts, maybe put it all in a central location instead?